Dockerfile

```
@@ -0,0 +1,21 @@
+FROM rustlang/rust:nightly AS builder
+
+WORKDIR /app
+
+COPY ./ ./
+
+RUN rustup default nightly && cargo build --release
+
+FROM debian:bookworm-slim
+
+RUN apt-get update && \
+  apt-get upgrade && \
+  apt-get install -y libsqlite3-0 libpq5 ca-certificates && \
```

```suggestion
  apt-get install -y ca-certificates && \
```
I don't think what we need those